lib/ansible/module_utils/fortios.py

```diff
@@ -0,0 +1,51 @@
+#
+# (c) 2017 Benjamin Jolivot, <bjolivot@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+import os
+import time
+
+
+fortios_argument_spec = dict(
+    host            = dict(required=True ),
+    username        = dict(required=True ),
+    password        = dict(required=True, type='str', no_log=True ),
+    timeout         = dict(type='int', default=60),
+    vdom            = dict(type='str', default=None ),
+    backup          = dict(type='bool', default=False),
+    backup_path     = dict(type='str'),
```

Comment:
`backup_path     = dict(type='path'),` would use  https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/basic.py#L1659 which allows to use `~` in the value, also see https://docs.python.org/3/library/os.path.html#os.path.expanduser